Makefile

```diff
@@ -1,9 +1,9 @@
-# VERSION defines the project version for the bundle.
+# OPVERSION defines the project version for the bundle.
 # Update this value when you upgrade the version of your project.
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
-# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
-# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 0.0.1
+# - use the OPVERSION as arg of the bundle target (e.g make bundle OPVERSION=0.0.2)
+# - use environment variables to overwrite this value (e.g export OPVERSION=0.0.2)
+OPVERSION ?= 0.0.1
```

Comment:
only concern here is VERSION is a fairly standard thing in our Makefile's. Would be nice not to change this

Reply:
pushed an alternative, @dprince  please check if it's ok this way or you think we need to handle it differently?